docs/source/tutorials/getting_started/compile.rst

```diff
@@ -46,6 +46,11 @@ In particular, OpenFPGA requires specific versions for the following dependencie
 
 :iverilog:
   version 10.1+ is required to run Verilog-to-Verification flow
+  
+:envyaml:
+  python package envyaml is required:
+  
+  python3 -m pip install envyaml
```

Comment:
you can replace this with `python3 -m pip install -r requirements.txt` 
all required packages are already listed in that, also it is tested on the CI.

Reply:
@ganeshgore Thanks for the input. @apond308 Can you update the file? We are good to go.

Reply:
Just changed it.